src/libpocketsphinx/.nn_mgau.c

```diff
@@ -0,0 +1,240 @@
+/* -*- c-basic-offset: 4; indent-tabs-mode: nil -*- */

```

Comment:
This file should not be added probably since you have the same file without the initial dot in the name

Reply:
I actually need this file because it has placeholder paths that are updated and copied into nn_mgau.c when autogen.sh is run.

Reply:
You do not need this file. In Linux software paths are usually hardcoded since filesystem layout is fixed. Check how model path is used for example.

Reply:
I'll try to see if I can get it to work or get back to you with the error.

Reply:
nn_mgau.c calls runNN.py which is present in the same folder. But when I put the path as "runNN.py" or "./runNN.py" I get a file not found.